de/figures/torre

```diff
@@ -0,0 +1,6 @@
+import numpy as np
+from ..importData import importRNAseqKO()
+from ..importData import formMatrix()
+
+
+    
```

Comment:
Do you need this file? ("torre.py")
If not, please delete the file from the figures folder.